include/ui/pixel-display-painted-item.hpp

```diff
@@ -0,0 +1,283 @@
+   * \param verticallyMirrored The verticallyMirrored to be set
+   */
+  void setVerticallyMirrored(bool verticallyMirrored);
+  /*
+   * \brief sets the tight
+   * \param tight The tight to be set
+   */
+  void setTight(bool tight);
+  /*
+   * \brief sets the pixelBufferPointerLike
+   * \param pixelBufferPointerLike The pixelBufferPointerLike to be set
+   */
+  void setPixelBufferPointerLike(bool pixelBufferPointerLike);
+  /*
+   * \brief sets the colorTablePointerLike
+   * \param colorTablePointerLike The colorTablePointerLike to be set
+   */
+  void setColorTablePointerLike(bool colorTablePointerLike);
+  /*
+   * \brief sets the freeBytes
+   * \param freeBytes The freeBytes to be set
+   */
+  void setFreeBytes(size_t freeBytes);
+  /*
+   * \brief sets the freeBits
+   * \param freeBits The freeBits to be set
+   */
+  void setFreeBits(size_t freeBits);
+
+  /*
+   * \brief returns pixelBaseAddress
+   * \returns pixelBaseAddress
+   */
+  size_t getPixelBaseAddress();
+  /*
+   * \brief returns colorBaseAddress
+   * \returns colorBaseAddress
+   */
+  size_t getColorBaseAddress();
+  /*
+   * \brief returns width
+   * \returns width
+   */
+  size_t getWidth();
+  /*
+   * \brief returns height
+   * \returns height
+   */
+  size_t getHeight();
+  /*
+   * \brief returns ColorMode
+   * \returns ColorMode
+   */
+  QString getColorMode();
+  /*
+   * \brief returns rBit
+   * \returns rBit
+   */
+  size_t getRBit();
+  /*
+   * \brief returns gBit
+   * \returns gBit
+   */
+  size_t getGBit();
+  /*
+   * \brief returns bBit
+   * \returns bBit
+   */
+  size_t getBBit();
+  /*
+   * \brief returns columns_rows
+   * \returns columns_rows
+   */
+  bool getColumns_rows();
+  /*
+   * \brief returns horizontallyMirrored
+   * \returns horizontallyMirrored
+   */
+  bool getHorizontallyMirrored();
+  /*
+   * \brief returns verticallyMirrored
+   * \returns verticallyMirrored
+   */
+  bool getVerticallyMirrored();
+  /*
+   * \brief returns tight
+   * \returns tight
+   */
+  bool getTight();
+  /*
+   * \brief returns pixelBufferPointerLike
+   * \returns pixelBufferPointerLike
+   */
+  bool getPixelBufferPointerLike();
+  /*
+   * \brief returns colorTablePointerLike
+   * \returns colorTablePointerLike
+   */
+  bool getColorTablePointerLike();
+  /*
+   * \brief returns freeBytes
+   * \returns freeBytes
+   */
+  size_t getFreeBytes();
+  /*
+   * \brief returns freeBits
+   * \returns freeBits
+   */
+  size_t getFreeBits();
+
+  /*
+   * \brief calls update and handles errors
+   */
+  void doUpdate();
+
+ private:
+
+  /*
+   * gives the index of a given name of a colorMode
+   * \param colorMode Name of the colorMode
+   */
+  static size_t stringToColorMode(const QString &colorMode);
+  /*
+   * gives the name of a given index of a colorMode
+   * \param colorMode Index of the colorMode
+   */
+  static QString colorModeToString(size_t colorMode);
+  /// shared pointer to the image
```

Comment:
Why do you use this style of commenting if the entire project never used this style? Write `/** ... */`